src/pages/App.tsx

```diff
@@ -88,6 +91,7 @@ export default function App() {
               <Route exact strict path="/pool" component={Pool} />
               <Route exact strict path="/png/:version" component={Earn} />
               <Route exact strict path="/vote" component={Vote} />
+              <Route exact strict path="/ido" component={IDO} />   // Added
```

Comment:
`// Added` is not valid comment in jsx
`{/* Added */}` is valid comment in jsx

Reply:
Removed invalid comments